<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmItem</class>
 <widget class="QWidget" name="FrmItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#FrmItem{border:1px solid rgb(220,220,220);}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>21</y>
        <width>181</width>
        <height>15</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(200, 200, 200);</string>
      </property>
      <property name="text">
       <string>2023-11-07 09:00:00</string>
      </property>
     </widget>
     <widget class="QLabel" name="txt_Depart">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>57</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>北京首都机场</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>63</x>
        <y>2</y>
        <width>69</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>机票</string>
      </property>
     </widget>
     <widget class="QLabel" name="txt_ArriveDate">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>77</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>2023-11-08 13:00:00</string>
      </property>
     </widget>
     <widget class="QLabel" name="txt_CraftType">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>37</y>
        <width>69</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>CA8504</string>
      </property>
     </widget>
     <widget class="QLabel" name="txt_Arrive">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>50</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>昆明长水机场</string>
      </property>
     </widget>
     <widget class="QLabel" name="txt_DepartDate">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>77</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>2023-11-08 10:00:00</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>32</y>
        <width>431</width>
        <height>10</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="txt_OrderStatus">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>0</y>
        <width>131</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 200, 0);</string>
      </property>
      <property name="text">
       <string>已完成</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>2</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/image/Resource/image/AirTicketBooking.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="txt_Price">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>50</y>
        <width>69</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>1400元</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
